__tests__/app.test.js

```diff
@@ -117,6 +118,115 @@ describe("app", () => {
         });
     });
   });
+  describe("PATCH /api/articles/:article_id", () => {
+    it("201: responds with updated article object, with correct properties", () => {
+      const testvote = { inc_votes: 1 };
+
+      return request(app)
+        .patch("/api/articles/1")
+        .send(testvote)
+        .expect(201)
```

Comment:
patch should expect a status code of 200, as it is updating rather than creating something :)